security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/ShuntCompensatorPositionActionBuilder.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.action;
+
+/**
+ * @author Miora Vedelago <miora.ralambotiana at rte-france.com>
+ */
+public class ShuntCompensatorPositionActionBuilder {
+
+    private String id;
+    private String shuntCompensatorId;
+    private Integer sectionCount = null;
+
+    public ShuntCompensatorPositionAction build() {
+        if (sectionCount == null) {
+            throw new IllegalArgumentException("sectionCount in undefined");
+        }
+        if (sectionCount < 0) {
```

Comment:
I think this action could lead to a zero section count, it is a way to easily disconnect the shunt compensator. NB: fix typo.